pkg/cli/adbtunnel.go

```diff
@@ -108,6 +117,40 @@ func openADBTunnel(flags *ADBTunnelFlags, c *command, args []string, opts *subCo
 	return merr
 }
 
+func listADBTunnels(flags *listADBTunnelFlags, c *command, args []string, opts *subCommandOpts) error {
+	controlDir := opts.InitialConfig.ADBControlDirExpanded()
+	entries, err := os.ReadDir(controlDir)
+	if err != nil {
+		return fmt.Errorf("Error reading control directory (%s): %w", controlDir, err)
```

Comment:
nit: %q will print a quotes string

Reply:
paths are not typically printed with quotes, changed the message to not need parenthesis either.